addon/components/search-dropdown/template.hbs

```diff
@@ -1,16 +1,15 @@
-<div class="osf-search" data-bind="fadeVisible: showSearch" style="display: none">
+<div class="osf-search">
     <div class="container">
         <div class="row">
             <div class="col-md-12">
-                <form class="input-group" data-bind="submit: submit">
-                    <input id="searchPageFullBar" name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search" data-bind="value: query, hasFocus: true">
+                <form class="input-group" {{action "search" on="submit"}}>
+                    <input id="searchPageFullBar" onchange={{action "updateQuery"}} name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search">
```

Comment:
You can use `value="target.value` here, and then update the signature of `updateQuery` to just take a value